app-admin/aerospike-amc-community/aerospike-amc-community-5.0.0.ebuild

```
@@ -0,0 +1,31 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="aerospike-amc-enterprise-${PV}-linux"
+
+DESCRIPTION="Web UI based monitoring tool for Aerospike Community Edition Server"
+HOMEPAGE="https://www.aerospike.com"
+SRC_URI="https://github.com/aerospike-community/amc/releases/download/${PV}/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	acct-group/aerospike-amc-community
+	acct-user/aerospike-amc-community
+"
+
+RDEPEND="${DEPEND}"
```

Is there a good reason not to just use `RDEPEND` directly? Is the user and group needed during installation?

```suggestion
RDEPEND="
	acct-group/aerospike-amc-community
	acct-user/aerospike-amc-community
"
```

---

Oh, i missed it, thanks